src/core/api.proto

```diff
@@ -94,6 +94,13 @@ message InferRequestHeader
     //@@       for tensors with a non-fixed-size datatype (like STRING).
     //@@
     uint64 batch_byte_size = 3;
+
+    //@@  .. cpp:var:: string shm_name
```

Comment:
Remove, this file is in the other commit, isn't it?

Reply:
It is needed to build and test successfully. I thought it would be ignored after I do a rebase.

Reply:
Rebased and overwritten.